code/modules/clothing/clothing.dm

```diff
@@ -632,8 +632,8 @@
 	//convenience var for defining the icon state for the overlay used when the clothing is worn.
 	//Also used by rolling/unrolling.
 	var/worn_state = null
-	valid_accessory_slots = list("utility","armband","decor","over")
-	restricted_accessory_slots = list("utility", "armband")
+	valid_accessory_slots = list("utility","armband","machete","decor","over") //VOREStation Edit - Adds Machete Sheath
+	restricted_accessory_slots = list("utility", "armband", "machete") //VOREStation Edit - Adds Machete Sheath

```

Comment:
The way you could make it noncompatible with holsters is to give holsters and machete holsters the same slot type.

If clothing has "X" in restricted_accessory_slots, then only one item with slot "X" can be an accessory at once. You can see here that "decor" and "over" are not restricted so you could have as many of those as you want.